src/AccessibilityInsights.Rules/Library/Structure/ControlView/Scrollbar.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Axe.Windows.Core.Bases;
+using Axe.Windows.Core.Enums;
+using Axe.Windows.Rules.PropertyConditions;
+using Axe.Windows.Rules.Resources;
+using System;
+using System.Globalization;
+using static Axe.Windows.Rules.PropertyConditions.ControlType;
+
+namespace Axe.Windows.Rules.Library
+{
+    [RuleInfo(ID = RuleId.ControlViewScrollbarStructure)]
+    class ControlViewScrollbarStructure : Rule
+    {
+        public ControlViewScrollbarStructure()
+        {
+            this.Info.Description = string.Format(CultureInfo.InvariantCulture, Descriptions.Structure, ControlView.ScrollbarStructure);
+            this.Info.HowToFix = string.Format(CultureInfo.InvariantCulture, HowToFix.Structure, ControlView.ScrollbarStructure);
+            this.Info.Standard = A11yCriteriaId.InfoAndRelationships;
+        }
+
+        public override EvaluationCode Evaluate(IA11yElement e)
+        {
+            if (e == null) throw new ArgumentException(nameof(e));
+
+            return ControlView.ScrollbarStructure.Matches(e) ? EvaluationCode.Pass : EvaluationCode.Note;
+        }
+
+        protected override Condition CreateCondition()
+        {
+            return ScrollBar;
```

Comment:
this rule only run when the ScrollBar control is marked with IsControlElementProperty true value. 
Please make sure that condition is added.

Reply:
I think this is abstracted inside the `ScrollBar` condition:
https://github.com/Microsoft/accessibility-insights-windows/blob/e455215a219a975d9ac05be7d25f37719a6e613a/src/AccessibilityInsights.Rules/PropertyConditions/ControlType.cs#L33

Reply:
I mistakenly thought the ScrollBar condition handled this. I will check with @RobGallo - it seems like none of the rules consider this property so we may need to fix it separately:
> Another such failing is that the children are not actually checked to make sure they have the IsContentElement or IsControlElement properties set to true. This failing is common throughout these rules. And again, there is no intent to remedy that as part of this change.
from #281

Reply:
tracking in #282